src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1405,6 +1447,8 @@ def test_ocl_apply():
 
     gen = str(psy.gen)
     assert "USE clfortran" in gen
+    # ATM no support for opencl compilation
```

Comment:
Is that true? Actually, I guess it is because OpenCL support in dl_esm_inf is not yet on master? If so, please could you create an issue for this and reference it here with a TODO.

Reply:
Now, that's a bit of an issue then: I've just added support for opencl compilation, and assumed the failures were caused by me not having an opencl compiler ... while it is apparently because of #316. Should I now go and remove --compileopencl support etc?? That would be quite a bit of work, which we would have to add back in again when doing #314. I've added a reference to #316 in this one case, but I am not sure what to do with then gocean1p0_opencl_test.py - should we comment out all those tests then?

Reply:
I'm happy to leave that flag in because, as you say, it's a lot of work and we will need it in the future.